packages/block-library/src/image/block.json

```diff
@@ -102,10 +102,10 @@
 		}
 	},
 	"selectors": {
-		"border": ".wp-block-image img"
-	},
-	"editorSelectors": {
-		"border": ".wp-block-image img, .wp-block-image .wp-block-image__crop-area"
+		"border": ".wp-block-image img, .wp-block-image .wp-block-image__crop-area",
+		"filter": {
+			"duotone": "img, .components-placeholder"
```

Comment:
Should this be `.wp-block-image img, .wp-block-image .components-placeholder`?

One of the ways the new stable API differs from the old experimental API is that it doesn't add the block wrapper to feature-level selectors ([see example](https://github.com/WordPress/gutenberg/pull/46496/files#r1150454241)). I wasn't able to test this PR yet, and I don't know if duotone would behave differently than any other feature-level selector in the old API. I thought I'd brought it up anyway given the tight schedule to ship this, in case anyone else can confirm/rebut.

Reply:
Looking into the Duotone support's use of these selectors, it appears as though Duotone support needs to be able to scope the selectors by a class generated from a filter id and slug which is applied on the block wrapper. 

We could make `wp_get_block_css_selector` and `getBlockCSSSelector` expect duotone selectors to be fully scoped. I believe this would work fine for global styles.

For the individual blocks with a duotone filter, we'll need to update `WP_Duotone_Gutenberg::render_duotone_support` to somehow add its custom duotone filter_id selector in at the correct place. I'm not sure if we can rely on either the standard block class or the root selector, in terms of being able to append the custom filter selector to either of them. The `filter.duotone` selector could be anything really.

Does that fit with your understanding @ajlende?

I think working through the kinks of the Selectors API in regard to Duotone support should be an immediate follow-up to this PR. Or we revisit the decision to revert the Selectors API after all.

Reply:
I've started scratching together a PR to bring duotone selectors more in line with other features but I'm still a little unsure on the best approach to resolve the questions above. 

Hopefully the code in https://github.com/WordPress/gutenberg/pull/49423 illustrates what I had in mind for the follow-up a little more.